Good container install but wont work after a container restart.

I installed the FreePbx docker as per this post: 

```
 docker run -d \
  --name freepbx \
  --restart=always \
  --net=host \
  --privileged \
  -v /docker/freepbx/certs:/certs \
  -v /docker/freepbx/www:/www/freepbx \
  -v /docker/freepbx/log:/var/log/ \
  -v /docker/freepbx/data:/data \
  -e DB_EMBEDDED=TRUE \
  tiredofit/freepbx
```

and it runs perfectly fine.
But if I restart the container for any reason - it doesnt seem to start back up.
The /admin page shows this error
```
Exception (1698)
SQLSTATE[HY000] [1698] Access denied for user 'freepbxuser'@'localhost'::SQLSTATE[HY000] [1698] Access denied for user 'freepbxuser'@'localhost'
```

Can I provide any more info that would help ?

@tiredofit I did what you mentioned but I got the following error
```
2018-09-25 13:09:05 0 [ERROR] InnoDB: The Auto-extending innodb_system data file './ibdata1' is of a different size 0 pages than specified in the .cnf file: initial 768 pages, max 0 (relevant if non-zero) pages!
2018-09-25 13:09:05 0 [ERROR] InnoDB: Plugin initialization aborted with error Generic error
2018-09-25 13:09:05 0 [Note] InnoDB: Starting shutdown...
```
it seems that the DB can't start

--- a/install/etc/cont-init.d/09-mariadb
+++ b/install/etc/cont-init.d/09-mariadb
@@ -11,7 +11,7 @@
   service mysql stop
   mkdir -p /var/lib/mysql
 
-     if [ ! -f /var/lib/mysql/mysql ]; then
+     if [ ! -d /var/lib/mysql/mysql ]; then
         echo '** [mysql] New Embedded Database Detected, setting up..'
         cp -R /assets/config/var/lib/mysql/* /var/lib/mysql
      fi

